quickwit-search/src/search_stream/mod.rs

```diff
@@ -34,14 +34,18 @@ use tantivy::fastfield::FastValue;
 /// Serialize the values into the `buffer` as bytes.
 ///
 /// Please note that the `buffer` is always cleared.
-pub fn serialize<TFastValue: FastValue + Display>(
+pub fn serialize<TFastValue: FastValue + Display, TPartitionValue: FastValue>(
```

Comment:
I would create a separated function for partition serialization where we also do the for loop and call then the `serialize`. That would avoid the `unwrap` here too.